templates/almalinux.yaml

```diff
@@ -1 +1 @@
-almalinux-9.yaml
\ No newline at end of file
+almalinux-10.yaml
```

Comment:
Probably we should bump up almalinux, rockylinux, oraclelinux, and centos-stream to 10 at once

Reply:
I don't mind reverting this part if needed for merge, but I don't think that's really a useful policy. All of those have somewhat different release cycles, especially CentOS Stream where 10 has been the "current" version for quite a while now, and Rocky and OL aren't even out with version 10 yet. Additionally, in practice people tend to choose one or two of them across infra, so there's no real utility to be gained from matching up all the versions.